src/components/inputselect/InputSelect.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const InputSelect = ({label}) => {
+  return(
+    <div className="input-group">
+      <label className="input-label">{label}</label>
+      <select name="" id="" className="input-select-field">
```

Comment:
this can be named as `input__select`

Reply:
Use BEM format where necessary. For example, all these form elements fall under FORM block, or in this case INPUT block. And `input-label` and `input-group` typically does not exist outside form.

Reply:
🆗